### ISSUE TITLE
Show out-of-date docs as a red badge

### ISSUE THREAD

**Issue by paf31** [purescript/pursuit#111]
If the docs are out-of-date with the Bower version, it would be nice if the badge were red. Is this possible? I guess it would need to query Bower or GitHub every time to find out, but maybe we can set Nginx to cache badges?

**Comment by hdgarrood** [comment]
Yes, we do currently cache badges. Invalidation would be slightly harder because the event that causes invalidation happens outside pursuit. I suppose we could have it check again if the badge is more than an hour old?

Also, if we do this: since we are using bower badges as well, perhaps it would make more sense to have the pursuit badge display the text "up to date" or "out of date" instead of the version number?

**Comment by paf31** [comment]
We probably don't have to worry about cache invalidation for now. A slight delay is fine.
